oak/server/rust/oak_runtime/src/lib.rs

```diff
@@ -568,18 +568,29 @@ impl OakNode {
 }
 
 /// Expected type for the main entrypoint to a Node under test.
-pub type NodeMain = fn(Handle) -> Result<(), oak::OakStatus>;
+pub type NodeMain = fn(Handle);
```

Comment:
I was wondering whether this could / should move to the `oak_abi` crate at some point?

Reply:
Yeah, I'd thought that too.  Added an extra commit.